firefoxThemes/waifus/faviconLoader.js

```diff
@@ -0,0 +1,25 @@
+import {buildSVG, svgToPng} from "../modules/utils/themes/logo.js";
+
+function setThemedFavicon(currentTheme) {
```

Comment:
The functions handling the favicon theming is located here!

Reply:
For the most part, the *Mixed Mode* favicon theming is live! 

## Issue
Except, if I try to **create a series of new tabs at the speed of light**, then the default favicon will be displayed for now onwards.

- [ ] Fix default favicon displaying after creating new tabs quickly

Reply:
:laughing: I just held down ctrl-t and was able to verify the issue. 
I'll just spin up a new issue once this is merged to address this. As this is probably not what users are normally going to do.